### ISSUE TITLE
Error: text color is not loaded properly in dark mode.

### ISSUE THREAD

**Issue by Huang-Zifeng** [djyde/cusdis#253]
The link to the website is https://www.huang-zifeng.com/

Please let me know if I could offer any details of the error.

![image](https://github.com/djyde/cusdis/assets/110339237/d20d8226-736d-4a4c-8013-61791b211d17)

**Comment by sunaiwen** [comment]
Try to set `data-theme` to `auto`.

**Comment by Huang-Zifeng** [comment]
Thanks, it does work. But it is kind of different with the demo shown on [cusdis.com](cusdis.com)(the background seems inconsistent). Can I just inverse the color of the text but the background remains? Thanks in advance for any advice given.
![image](https://github.com/djyde/cusdis/assets/110339237/5f158387-a9b7-434a-a265-96c46b7e1e90)

**Comment by sunaiwen** [comment]
> Thanks, it does work. But it is kind of different with the demo shown on [cusdis.com](cusdis.com)(the background seems inconsistent). Can I just inverse the color of the text but the background remains? Thanks in advance for any advice given. ![image](https://user-images.githubusercontent.com/110339237/257398737-5f158387-a9b7-434a-a265-96c46b7e1e90.png)

Hi, I happened being trapped on the same problem. I found it's due the `dark` class on a div. Remove the statement `class:dark={theme === 'dark'}` [here](https://github.com/djyde/cusdis/blob/master/widget/Widget.svelte#L95) solved my problem. But I still have no idea where this dark grey background comes from. 🥲

**Comment by Huang-Zifeng** [comment]
Thank you! But it seems that I will need to deploy cusdis myself to modify the html element, make it a little inconvenient for [hosted-service](https://cusdis.com/dashboard) user. Hope someone could PR it. Anyway, this issue is fixed for me, thanks again~

**Comment by cushyno** [comment]
This issue tripped me up on the hosted service too. I identified the problem to be the stylesheet (style.css) specifying among other styles, this type of CSS:
`.dark .dark\:text-gray-100 {
  --tw-text-opacity: 1;
  color:rgba(243,244,246,var(--tw-text-opacity))
}`

This requires that the tags with class "dark: text-gray-100" be descendants of tags with class "dark". However, the srcdoc html doesn't have any elements with class "dark".  So for me, auto mode doesn't work. 

I resolved this by injecting the corrected CSS using a hack similar to [Barth](https://barthpaleologue.github.io/Blog/posts/add-and-customize-cusdis-hugo/), though the style replace was handled with a page load trigger (delayed) rather than the mutation observer.
